From f5bd553dfd8c423fa8474312f929a727d1cd2633 Mon Sep 17 00:00:00 2001
From: Anton Mikanovich <amikan@ilbers.de>
Date: Wed, 14 Jul 2021 09:52:32 +0300
Subject: [PATCH 4/7] buildchroot: Introduce buildchroot dependency variable

To allow setting dependencies on proper BUILDCHROOT based on
ISAR_CROSS_COMPILE value we need to save propriate build task in
datastore variable to use it later in task recipes.

Signed-off-by: Anton Mikanovich <amikan@ilbers.de>
---
 meta/classes/buildchroot.bbclass | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meta/classes/buildchroot.bbclass b/meta/classes/buildchroot.bbclass
index 806a29f..0db6ef9 100644
--- a/meta/classes/buildchroot.bbclass
+++ b/meta/classes/buildchroot.bbclass
@@ -18,6 +18,7 @@ python __anonymous() {
         rootfs = d.getVar('BUILDCHROOT_HOST_DIR', True)
 
     d.setVarFlag('do_apt_fetch', 'depends', dep)
+    d.setVar('BUILDCHROOT_DEP', dep)
     d.setVar('BUILDCHROOT_DIR', rootfs)
 }
 
-- 
2.20.1

